docs/DeveloperGuide.md

```diff
@@ -35,4 +212,21 @@
 
 ## Instructions for manual testing
 
-{Give instructions on how to do a manual product testing e.g., how to load sample data to be used for testing}
+### Adding a student to the student list
```

Comment:
![image](https://github.com/nus-cs2113-AY2324S2/tp/assets/88656341/2fd2fef9-4662-43a7-9e52-5d0342e7a2f2)
This format seems abit cluttered, could use screenshots to help with visualisation